buildspec_sync.yml

```diff
@@ -2,7 +2,7 @@ version: 0.2
 phases:
   install:
     runtime-versions:
-      python: 3.7
+      python: 3.x
```

Comment:
I think this doesn't work in gov cloud or china... I think we've tried this before... unless something has changed... let's check the git blame

Reply:
See here: https://github.com/aws/aws-for-fluent-bit/commit/edbbb2c92f1bb495da45f57d538c84d6858e1ae8

Reply:
I doubt they have changed it